go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/intel/cri-resource-manager
 
-go 1.21
+go 1.22
```

Comment:
since Go 1.21 this means the minimum required go version which follows the new versioning. you probably want 1.22.0 here (see the codeql warnings also)

Reply:
Good point! See #1105